### PR TITLE
Delete unnecessary wordings in Handling GET message with no-body

### PR DESCRIPTION
**Pull request by arunans23**
Resolves https://github.com/wso2/docs-ei/issues/1750
Porting https://github.com/wso2/docs-ei/pull/2602

### CODE REVIEW

**Review by nilminiwso2 — APPROVED** [review]

File: en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md [hunk 1]
```diff
@@ -1,24 +1,19 @@
-# Using GET with a Message Body
-Typically, a GET request does not contain a body, and the Micro Integrator does not support these types of requests. When it receives a GET request that contains a body, it drops the message body as it sends the message to the endpoint. 
+# GET request with a Message Body
+Typically, a GET request does not contain a body, and the Micro Integrator will not consume the payload even if there is one.
 
 ## Synapse configuration
 
-Following is a sample REST API configuration that we can used to implement this scenario. See the instructions on how to [build and run](#build-and-run) this example.
+Following is a sample REST API configuration that we can use to validate this scenario. See the instructions on how to [build and run](#build-and-run) this example.
```
Comment (bhathiya) [inline en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md:6]: why does a user need to validate this? what's the use case? @nilminiwso2

Comment (bhathiya) [reply en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md:6]: @arunans23

Comment (nilminiwso2) [reply en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md:6]: @bhathiya This [1] is how we had this content in previous versions [1]. When we moved the content to 7.x, we used a consistent format with samples that users can try out.

I get your point about 'Why do users need to validate this'. However, currently, this doc serves the purpose of demonstrating how 'GET requests with a message body' is processed in the MI. 
If this is a limitation, @arunans23 - we can add a note that it is a limitation or workaround.

[1] https://docs.wso2.com/display/EI660/Configuring+Specific+Use+Cases+

Comment (bhathiya) [reply en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md:6]: I'm talking purely as a user here. I in fact came to this doc while searching on how to send a payload with a GET in EI. After reading the 1st para, I got to know it's not supported. I don't think I needed a demonstration of something not working. Are there any use cases you can think of where this content will be useful to someone?

Comment (arunans23) [reply en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md:6]: All three samples under the Handling special cases section does not serve any real purpose. Maybe it is better to remove them all and consolidate the three docs into one.

Comment (bhathiya) [reply en/micro-integrator/docs/use-cases/examples/rest_api_examples/using-get-with-a-body.md:6]: +1

**Pull request merged**